tableau_utilities/scripts/cli.py

```diff
@@ -329,6 +336,86 @@ def tableau_authentication(args):
     return t
 
 
+def set_datasource_connection_args(args):
```

Comment:
This basically follows the same pattern as the `tableau_authentication` function above.

Reply:
Ok. I can test this later. Let's try to do all that merge history stuff and then move forward.

I don't have strong feelings about this being in a separate file or not.

Reply:
Sounds good, thanks Jay! :)